### DISCUSSION TITLE
MacOS 13 Ventura compatability

### DISCUSSION THREAD

**Discussion by compulsivebuyer** [Ascoware/get-iplayer-automator#427] [Q&A]
Apologies if this has already been asked but my search returned no results: Is latest GIA compatible with Ventura?

**Comment by skovatch** [comment]
Yes, I build GiA on Ventura and run it daily. What are your cache settings? What did you search for?

**Comment by JohnN6** [comment]
Get Iplayer Automator does not seem to work properly on Ventura for me.  It loads correctly and updates caches.  I can skit to use a page from my browser - all exactly as normal - but when I click start, the download ribbon appears and never moves.   No entries are made in the directory where downloads are meant to go.   So currently Ventura 13.1 does not seem to run G.I.A. 1.24.4 on either Apple Silicon or on an Intel based machine.

**Comment by skovatch** [comment]
Check for empty lines in the download queue. If they are there, delete and try again.

**Comment by JohnN6** [comment]
Thanks .  There was a blank line which , of course, does not show up as obvious.   Perhaps a check for this in the next GUI version will be worthwhile.   i will report the cure to my local conference board where others had noted the same problem.
John N.
